frontend_server/lib/server.dart

```diff
@@ -112,14 +113,13 @@ abstract class CompilerInterface {
 class BinaryPrinterFactory {
   /// Creates new [BinaryPrinter] to write to [targetSink].
   BinaryPrinter newBinaryPrinter(IOSink targetSink) {
-    return new LimitedBinaryPrinter(
-      targetSink,
-      (_) => true /* predicate */,
-      false /* excludeUriToSource */);  }
+    return new LimitedBinaryPrinter(targetSink, (_) => true /* predicate */,
+        false /* excludeUriToSource */);
```

Comment:
this one isn't egregious but you could also get dartfmt to do better by putting a comma at the end of the argument list, i think.